test/integration/blocks-raw-handling.spec.js

```diff
@@ -58,20 +58,20 @@ describe( 'Blocks raw handling', () => {
 			HTML: '&lt;p&gt;Some &lt;strong&gt;bold&lt;/strong&gt; text.&lt;/p&gt;',
 			plainText: '<p>Some <strong>bold</strong> text.</p>',
 			mode: 'AUTO',
-		} ).map( getBlockContent ).join( '' );
+		} );
 
-		expect( filtered ).toBe( '<p>Some <strong>bold</strong> text.</p>' );
+		expect( filtered ).toBe( 'Some <strong>bold</strong> text.' );
 		expect( console ).toHaveLogged();
 	} );
 
 	it( 'should parse Markdown with HTML', () => {
 		const filtered = rawHandler( {
 			HTML: '',
-			plainText: '# Some <em>heading</em>',
+			plainText: '# Some <em>heading</em>\n\nA paragraph.',
```

Comment:
Why has this test changed?

Reply:
Because this specific test tests Markdown parsing, which together with this PR would only result into a heading if there's multiple lines paste.